lib/node.js

```diff
@@ -8,19 +8,45 @@ const UILabelClass = ObjC.classes.UILabel.class();
 const UITextFieldClass = ObjC.classes.UITextField.class();
 const UITextViewClass = ObjC.classes.UITextView.class();
 let UIWebViewClass = null;
+let WKWebViewClass = null;
 const UIWindow = ObjC.classes.UIWindow;
 const UIWindowClass = UIWindow.class();
 
 const textField_shouldChangeCharactersInRange_replacementString_ = ObjC.selector('textField:shouldChangeCharactersInRange:replacementString:');
 
+const activeInstances = {};
+
+const willMoveToWindow = ObjC.classes.UIView['- willMoveToWindow:'];
+const originalWillMoveToWindow = willMoveToWindow.implementation;
+willMoveToWindow.implementation = ObjC.implement(willMoveToWindow, (handle, selector, window) => {
+  originalWillMoveToWindow(handle, selector, window);
+
+  const key = handle.toString();
+
+  if (window.isNull() && key in activeInstances) {
+    const view = activeInstances[key].instance;
+    activeInstances[key].instance = null;
```

Comment:
I guess we can merge the `in` and `activeInstances[key]` lookups into one by assigning to a local variable and checking if it's different from `undefined`. That way we turn three lookups into one.

Reply:
cool! also moved that to a function so it's lazy